### PR TITLE
Chat Page Beginneth

### PR DESCRIPTION
**Pull request by LarrySchultheis**
Added basic chat page. Currently non-functional but match population works. Added cypress test for message page. Draft to link to in doc.

### CODE REVIEW

**Comment by LarrySchultheis** [comment]
Will fix outstanding compiler issue next week. Just going to overwrite merge checks

File: src/App.js [hunk 1]
```diff
@@ -17,31 +17,34 @@ import AdminManagement from './components/AdminManagement';
...
-        email: user.email
-      }
-      }).then(response => {
-        setUserRole(response?.data["role"]);
-        setUserObj(response?.data["user_obj"]);
-      });
+      const requestOptions = {
+        method: 'GET',
+      };
+      fetch(`http://localhost:5000/get_user_role?id=${user.sub}&email=${user.email}`, requestOptions)
```
Comment (squaill) [inline src/App.js:34]: we should touch base about this tomorrow too. I'm confused about the reason for these changes

File: src/components/ChatService/ChatManager.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { Socket } from "./Socket.js"
+import React, {useEffect, useState} from "react"
+
+export default function ChatManager() {
+    const [message, setMessage] = useState("");
+    const handleResponse = (data) => {
+        console.log(data)
+    }
+    const sendMessage = () => {
+        console.log('sending message');
```
Comment (squaill) [inline src/components/ChatService/ChatManager.js:11]: are these console logs an oops?

File: src/components/ChatService/ChatPage.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    if (a.ts < b.ts)
+       return -1;
+    if (a.ts > b.ts)
+      return 1;
+    return 0;
+  }
+    
+  useEffect(() => {
+    if (props.user) {
+      axiosInstance.get(`/matches?email=${props.user.email}`).then(response => {
```
Comment (squaill) [inline src/components/ChatService/ChatPage.js:32]: 🎉🎉🎉🎉🎉🎉 thx for using axios

**Review by squaill — CHANGES_REQUESTED** [review]

**Comment by LarrySchultheis** [comment]
I think its good enough

**Pull request merged**